addons/binding/org.openhab.binding.camera/README.md

```diff
@@ -0,0 +1,49 @@
+# Camera Binding
+
+The binding integrates simple IP base cameras.  
+
+Almost every IP camera has a simple HTTP access API defined by Urls to access at least static images.
+Additionally video streams like H264.x oder MJPEG and others are very wide spread. This binding supports these types of cameras.
+
+-- Thomas Hartwig 2016-07-24
+
+## Supported Cameras
+
+All cameras supporting simple HTTP based API fetching for JPG/PNG images.
+Please contact the author Thomas Hartwig <thomas.hartwig@gmail.com> for camera Url examples if not listed here.
+
+Vendor | Single snapshot URL | Video capture (not yet supported) |
+:------:|:-------------------:|:---------------------------------:|
+ Axis | http://<CAMERA>/jpg/image.jpg
+
+## Discovery
+
+Currently camera discovery is not supported, probably ONVIF might supported soon.
+
+## Thing Configuration
+
+* self explaining when devices are created in OpenHab
+
+
+## Channels
+
+### image
+
+This is the base image channel which is updated periodically in the configured interval. This is currently the only supported channel.
+
+## Near Future
```

Comment:
I think you can remove this from the readme, the goal of the readme is different.

Reply:
OK, moved it to a TODO file.